### DISCUSSION TITLE
CDDB v 117

### DISCUSSION THREAD

**Discussion by rcsauce** [enzo1982/freac#608] [Help & Support]
Hi - I recently updated and I am getting an error message from the CDDB database.  Please send me the proper configuration.

**Comment by ohquait** [comment]
"Options"
"General settings setup"
..........
"CDDB"
X Enable remote CDDB database
CDDB access mode:  HTTP POST
CDDB server:             gnudb.gnudb.org     Port:  80
eMail addresss:          cddb@freac.org

is it?

**Comment by SquigglyDog** [comment]
CDDB does not seem to function on FREAC or other rippers(ie Asunder)
